Automattic/wp-calypso
# Inline Help: extract Popover content to separate component

This is one of the crucial parts of #55189: extracting the Inline Help popover content from the actual `Popover` component. This content component will be rendered, without the parent `Popover` component, in the Inline Help widget.

**How to test and review:**
Verify that all needed props are passed between the `Popover` and `Content` components. Verify that the split of the SCSS stylesheet into two is done correctly. In `style.scss`, there remain only styles for the round question mark button and for the popover positioning, and everything about the content is moved the the `popover-content.scss` module.

There are two commits that remove unused CSS styles: for switching between Classic and Block editor, and for a notification dot on the help button that used to be used when Site Checklist was integrated with the Inline Help widget.
